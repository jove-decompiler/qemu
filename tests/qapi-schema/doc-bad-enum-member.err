doc-bad-enum-member.json:3: the following documented members are not in the declaration: a
