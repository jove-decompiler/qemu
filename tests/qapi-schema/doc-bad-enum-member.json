# Members listed in the doc comment must exist in the actual schema
# BUG: nonexistent @a is not rejected

##
# @Foo:
# @a: a
# @b: b
##
{ 'enum': 'Foo', 'data': [ 'b' ] }
