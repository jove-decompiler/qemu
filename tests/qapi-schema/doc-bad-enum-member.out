module None
object q_empty
enum QType
    prefix QTYPE
    member none
    member qnull
    member qnum
    member qstring
    member qdict
    member qlist
    member qbool
module doc-bad-enum-member.json
enum Foo
    member b
doc symbol=Foo
    body=

    arg=a
a
    arg=b
b
